Package/Captcha/Captcha.php

```diff
@@ -255,9 +263,19 @@ public function html()
         if (!$this->getPublicKey()) {
             throw new Exception('You must set public key provided by reCaptcha');
         }
+        
+        /**
+         * JavaScript resource (api.js) parameters
+         * @see https://developers.google.com/recaptcha/docs/display#js_param
+         */
+        $jsParams = array();
+        if (!is_null($this->languageCode)) {
+            $jsParams['hl'] = $this->languageCode;
+        }
+        $jsParamsQuery = !empty($jsParams) ? '?'.http_build_query($jsParams) : '';
```

Comment:
@armezit The http_build_query could return something else in future / previous PHP versions this is simply a safe guard for possible revisions to PHP core. Also '?'.http should be '?' . http_

Thank you!